remote.lua

```diff
@@ -33,12 +33,18 @@ end
 
 --
 -- Creates a sensor managed by a remote interface (another mod or script)
+-- mod_name: Name of the mod registering the sensor. Sensor will be removed if the mod is removed from the game.
 -- sensor_name: internal name of the sensor. Should be unique.
 -- sensor_text: Text to display in the active gui
 -- sensor_caption: Sensor setting name in the EvoGUI settings panel
 -- sensor_color: Font color of the text to display in the active gui, optional, may be nil
 -- example: remote.call("EvoGUI", "create_remote_sensor", "mymod_my_sensor_name", "Text: Lorem Ipsum", "[My Mod] Lorem Ipsum Text")
-local function create_remote_sensor(sensor_name, sensor_text, sensor_caption, sensor_color)
+local function create_remote_sensor(mod_name, sensor_name, sensor_text, sensor_caption, sensor_color)
+    if not mod_name then
+        evogui.log({"err_nomodname"})
```

Comment:
If you move this a few lines down, you can include the sensor name in the error message, which should be easier to understand for readers.